testbed/tests/metric_test.go

```diff
@@ -33,8 +33,8 @@ func TestMetric10kDPS(t *testing.T) {
 			NewSFxMetricDataSender(testbed.GetAvailablePort(t)),
 			NewSFxMetricsDataReceiver(testbed.GetAvailablePort(t)),
 			testbed.ResourceSpec{
-				ExpectedMaxCPU: 48,
-				ExpectedMaxRAM: 52,
+				ExpectedMaxCPU: 40,
```

Comment:
I'm not sure what the new baseline for CPU should be. It doesn't exceed 25 on both CI and my local machine but limit listed here was 48. I decided to be a bit conservative and reduce it to 40.

Reply:
I typically set it to observed max * 1.3